pass/__init__.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+
+import fnmatch
+import os
+from albert import *
+
+md_iid = "0.5"
+md_version = "0.5"
```

Comment:
Make the version 1.2 before the interface id and plugin version has been one. 0.4 was the interface '.1' was the plugin version. Now it is iid 0.5 and version (increased) 1.2